lib/Doctrine/Table.php

```diff
@@ -1132,6 +1130,11 @@ public function processOrderBy($alias, $orderBy, $columnNames = false)
            $alias = $this->getComponentName();
         }
 
+        // Php8.1 require a string
+        if (null === $orderBy) {
```

Comment:
Yoda sytax again
```suggestion
        if ($orderBy === null) {
```

Reply:
I'm still hesitant to modify this kind of thing.